script.adzapper/resources/lib/settings.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove all the .pyc files from the submission. Add __pycache__ to gitignore